README.md

```diff
@@ -1,2 +1,3 @@
 # fiware-health
 Program to check the functionality of each FIWARE Lab node.
```

Comment:
It is depend that what are you doing in that directory. I mean, if this program it itself a sanity check then you should move the content to the main menu.

Anyway, I suppose that you execute this code like a test and not like a python program, is it correct? If the answer is true, then the description that you put in the subdirectory should be move to the main directory due to you do not execute the program in different way.

Nevertheless, if you have two separate way to execute the program (python xxx.py and nosetest for execution of the test) then we would think about any different REAME.rst files.